src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/TdsParserStateObject.netcore.cs

```diff
@@ -393,60 +401,9 @@ private void ReadSniError(TdsParserStateObject stateObj, uint error)
             AssertValidState();
         }
 
-        public void ProcessSniPacket(PacketHandle packet, uint error)
```

Comment:
Can we still keep the older function? I am still trying to understand all the checks that were removed with the new function call.

Reply:
The previous version of `ProcessSniPacket` ? Why would we? we can't swap out the implementations without also adding the ability to disable all partial packet checking. The core idea of the PR was that the change was limited to a small location, primarily `ProcessSniPacket` which is the communication point between two layers. Unfortunately the rather messy bug fixes that have been needed in other places make this more complex but `ProcessSniPacket` is still the primary location where new functionality is present.